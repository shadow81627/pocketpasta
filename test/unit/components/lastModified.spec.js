import Component from '@/components/Layout/lastModified.vue';
import { shallowMount, createLocalVue } from '@vue/test-utils';
import Vuex from 'vuex';
import Vuetify from 'vuetify';
import Vue from 'vue';

const localVue = createLocalVue();
let vuetify;

localVue.use(Vuex);
Vue.use(Vuetify);

const factory = () =>
  shallowMount(Component, {
    localVue,
    vuetify,
    mocks: { $config: {} },
    propsData: {
      utc: true,
    },
  });

describe('Last Modified', () => {
  beforeEach(() => {
    vuetify = new Vuetify();
  });
  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.vm).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
