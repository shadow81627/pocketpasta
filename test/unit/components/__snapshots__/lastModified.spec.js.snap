// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Last Modified renders properly 1`] = `
"<span><v-col-stub cols=\\"auto\\" tag=\\"div\\"><v-card-stub loaderheight=\\"4\\" tag=\\"div\\" color=\\"transparent\\" tile=\\"true\\" flat=\\"true\\"><v-card-text-stub class=\\"py-2\\"><span>Last modified: </span> <span>Invalid Date</span></v-card-text-stub>
</v-card-stub>
</v-col-stub></span>"
`;
